src/directions.common.ts

```diff
@@ -42,6 +42,8 @@ export interface NavigateToOptions {
      */
     allowGoogleMapsWeb?: boolean;
   };
+
+  type?: string;
```

Comment:
It's even nicer to do this: `type?: "driving" | "transit" | "bicycling" | "walking";`

I'll change that after merging.

Reply:
ah yeah ofc, and then its possible to throw a error on that